source/assets/public/index.css

```diff
@@ -72,25 +74,29 @@ body {
 .breakpoint-desktop-up .container {
   padding-left: 15rem;
 }
+
+/* Navigation */
+
+.nav {
+  line-height: 3rem;
+  padding-bottom: 1.5rem;
+  padding-right: 1rem;
+  text-align: right;
+}
+  .nav__item {
+    color: var(--primary-color);
+    display: inline;
+    font-family: var(--font-subtitle);
+    font-size: 1.6rem;
+    padding-left: 4rem;
+    text-decoration: none;
+  }
```

Comment:
Yeah sorry that's just to help me see what belongs to what. You're welcome to change it if it makes more sense to you the other way! :)